Contributors/adwaitbytes.md

```diff
@@ -0,0 +1,85 @@
+# 🚀 Adwait Keshari
+
+🎯 **Aspiring Entrepreneur | Developer | Data Analyst**  
+🌐 **Passionate about Open Source, Web3, and Data-Driven Innovation**
+
+## 🏫 Education
+
+- **Bachelor of Science in Programming and Data Science**  
+  *Indian Institute of Technology Madras (IITM)*  
+  *2023 - Present*  
+  Pursuing a comprehensive program focusing on programming, data analysis, and computational thinking.
+
+## 🏆 Achievements
+
+- **Content and Media Manager at Aarambh Labs**  
+  Spearheading content strategy and media outreach to promote innovative projects.
+
+- **Secured 5th position on IIT Madras Hackathon**
+  Secured 5th postion on the Hacksprint Hackathon Organised by IIT Madras.
+
+- **City Coordinator at IITM Boundless Club - Bhopal**  
+  Coordinated events and initiatives, fostering community engagement and learning.
+
+- **Conducted Arduino Workshop for IITM Sahyog Club**  
+  Led hands-on sessions, enhancing participants' understanding of hardware programming.
+
+- **Selected for Accenture Innovation Challenge 2024**  
+  Recognized for innovative solutions and problem-solving skills in a competitive environment.
+
+
+## 🔥 Skills & Tech Stack
+
+### **Programming**
+
+- **Languages**: Python 🐍 | C++ ⚡ | SQL 🗄️  
+- **Concepts**: Data Structures & Algorithms
+
+### **Tools & Frameworks**
+
+- **Version Control**: Git & GitHub  
+- **Documentation**: Markdown  
+- **Operating Systems**: Linux, Windows
+
+### **Web3 & Blockchain**
+
+- **Smart Contracts**: Solidity  
+- **Blockchain Platforms**: Ethereum, Solana, Blinkord  
+- **Decentralized Applications (dApps)**
+
+## 📊 Projects
+
+### 🏥 Medical Store Sales Analysis
+
+- **Objective**: Analyze daily sales and customer behavior in a medical shop.  
+- **Outcome**: Identified patterns affecting revenue, leading to strategic improvements.
+
+### 🗳️ Election Data Scraper
+
+- **Objective**: Develop a web scraper to extract real-time election data.  
```

Comment:
what was the project, what difficulties you faced, what did you learn and outcome should be empirical (x% efficient in scraping real time data than products available etc)